funds_api.py

```diff
@@ -0,0 +1,71 @@
+import BaseHTTPServer
```

Comment:
This is Python2? Pretty sure stripe works with python3, is there a reason for not using it?
https://docs.python.org/3.5/library/http.html#module-http

https://github.com/stripe/stripe-python
> We commit to being compatible with Python 2.6+, Python 3.3+ and PyPy